src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void drawFigure();
```

Comment:
```suggestion
    void draw();
```
Этот интерфейс могут имплементировать разные классы, не только Figure. И тогда метод drawFigure будет выглядеть странно. Помни про Абстракцию.